OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/preferencecenter/PreferenceUpdate.java

```diff
@@ -0,0 +1,21 @@
+package com.optimove.android.preferencecenter;
+
+import java.util.List;
+
+public class PreferenceUpdate {
+    private final String id;
```

Comment:
here since we dont have TopicId type like in web, mb worth calling it `topicId` -- to make it more clear which id we need. Otherwise nothing here points to topicId. 

We might later update web to stay consistent if have to